store/postgres/src/relational_queries.rs

```diff
@@ -492,17 +492,14 @@ impl<'a> QueryFragment<Pg> for QueryValue<'a> {
                     }
                     // TSVector will only be in a Value::List() for inserts so "to_tsvector" can always be used here
                     ColumnType::TSVector(config) => {
-                        out.push_sql("(");
+                        out.push_sql("''::tsvector");
```

Comment:
Two quick comments:

1. Since this can be used in a larger query, can we keep the whole construct enclosed in parentheses?
2. I assume this is to deal with the case where `sql_values` is empty, right? It would be clearer if the code was structured as
```
if sql_values.is_empty() {
  out.push_sql("''::tsvector");
} else {
  .. what we did before this PR ..
}
```